pkg/porter/explain.go

```diff
@@ -115,10 +115,16 @@ func (s SortPrintableAction) Swap(i, j int) {
 }
 
 func (o *ExplainOpts) Validate(args []string, cxt *context.Context) error {
-	err := o.sharedOptions.Validate(args, cxt)
+	err := o.validateInstanceName(args)
```

Comment:
What's odd here is that explain really shouldn't be using the shared options... That is meant for the actions, like install and upgrade. Thanks for fixing.